aioconsole/stream.py

```diff
@@ -159,9 +159,12 @@ async def open_stantard_pipe_connection(pipe_in, pipe_out, pipe_err, *, loop=Non
     return in_reader, out_writer, err_writer
 
 
+open_stantard_pipe_connection = open_standard_pipe_connection
```

Comment:
There's no need for backward compatibility here

Reply:
Can you use a suggestion? Then I can apply it as I'm afk